src/main/java/duke/model/events/Event.java

```diff
@@ -0,0 +1,61 @@
+package duke.model.events;
+
+import duke.commons.exceptions.DukeException;
+import duke.logic.api.ApiParser;
+import duke.model.locations.Venue;
+
+import java.time.LocalDateTime;
+
+public class Event extends DoWithin {
+    private Venue venue;
+
+    /**
+     * Initializes a event task with location.
+     *
+     * @param locationDescription A description of this task.
+     * @param startDate Starting date of task
+     * @param endDate Ending date of task
+     */
+    public Event(String locationDescription, LocalDateTime startDate, LocalDateTime endDate) throws DukeException {
+        super(locationDescription, startDate, endDate);
+        // This can be removed once we implement the map ?
```

Comment:
use comments on Github (like this one) to convey message to the other developers (or yourself at a later time) rather than messages like this

Reply:
Code deleted